change/@fluentui-eslint-plugin-4067e75c-636a-4ca2-a360-3d1e3e43a9bb.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat(eslint-plugin): enable no-export-all:all for all packages public API",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "martinhochel@microsoft.com",
+  "dependentChangeType": "patch"
```

This should be none to prevent bumping all packages.
```suggestion
  "dependentChangeType": "none"
```

---

ah great catch! I forget this all the time. cannot this be handled by CLI somehow ? or cases like this need to be handled "manually" by us ?

---

It would be simple to add as a beachball option--I made an issue for this awhile back. https://github.com/microsoft/beachball/issues/582

This person also mentions that it's already possible to set `dependentChangeType` to `none` in `beachball.config.js` (probably an unintended side effect of object spreading), but I recently tried putting `dependentChangeType` in eslint-plugin's package.json under `beachball` and that didn't work.
https://github.com/microsoft/beachball/issues/605